jujugui/static/gui/src/app/d3-components/d3-components.js

```diff
@@ -162,25 +162,25 @@ YUI.add('d3-components', function(Y) {
     _normalizeHandler: function(handler, module, selector) {
       var result = {};
 
-      if (L.isString(handler)) {
+      if (typeof(handler) === 'string') {
         result.callback = module[handler];
         result.phase = 'on';
       }
 
-      if (L.isObject(handler)) {
+      if (typeof(handler) === 'object' && handler !== null) {
```

Comment:
@Lukewh Do you think we need the utils when the built in check works? Most of the checks e.g. typeof('string') === 'string' don't need to do anything more than check the type, and I wonder if the util gives us anything?